pylintrc

```diff
@@ -321,13 +321,13 @@ max-statements=50
 max-parents=7
 
 # Maximum number of attributes for a class (see R0902).
-max-attributes=7
+max-attributes=8
```

Comment:
Xavier does not want us to change these limits, nor to add pylint overrides to the code base.  I'm not sure what to do, since to me the code looks good.

Using the single structured `domains` attribute I suggested above would avoid hitting this limit, but I don't think it's a good idea to use it before we can use proper JSONB fields.

CC @antoviaque